V1 timetracking

maartenscholz opened the pull request.

steftrenson commented on src/apiary.apib:
```diff
@@ -17,6 +17,8 @@ This is the Teamleader API v2 documentation.
 :[Files](files.apib)
 
-:[Meetings](meetings.apib)
+:[Meetings](meetings.apib
```
Missing )

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+
++ Response 200 (application/json)
+
+    + Attributes
```
Missing backticks for the attribute examples (used in previous examples)

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
++ Request (application/json)
+
+    + Attributes
+        + task_id: 1 (number)
```
missing backticks in example

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+                + MC
+                + VCMD
+        + product_id_1: `1` (number, optional) - Id of the product for the additional cost
+        + subtitle_1: subtitle (string, optional) - Additional cost subtitle
```
missing backticks for the example value

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+                + none
+                + task
+        + related_object_id: `1` (number, optional) - If this activity is related to a task, pass in the task ID
+        + description_1: description (string, optional) - Additional costs description
```
missing backticks for the example value

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+        + end_date: `1498048402` (number, optional) - End date of the activity as Unix timestamp (seconds)
+        + worker_id: `12` (number, optional) - Id of the user
+        + task_type_id: `13` (number, optional) - Type id of the task
+        + for: company (enum[string], optional) - Who this activity should be billed to
```
missing backticks for the example value

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+             + Members
+                 + 0
+                 + 1
+        + related_object_type: task (enum[string], optional) - What kind of item this activity is related to
```
missing backticks for the example value(s)

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+
+    + Attributes
+        + timetracking_id: `1` (number) - ID of the timetracking to update
+        + description: description (string, optional) - Description of the activity
```
missing backticks for the example

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+                + MC
+                + VCMD
+        + product_id_1: `1` (number, optional) - Id of the product for the additional cost
+        + subtitle_1: subtitle (string, optional) - Additional cost subtitle
```
missing backticks in the example value

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+                + none
+                + task
+        + related_object_id: `1` (number, optional) - If this activity is related to a task, pass in the task ID
+        + description_1: description (string, optional) - Additional costs description
```
missing backticks

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+             + Members
+                 + 0
+                 + 1
+        + related_object_type: task (enum[string], optional) - What kind of item this activity is related to
```
missing backticks for the example value(s)

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+        + end_date: `1498048402` (number) - End date of the activity as Unix timestamp (seconds)
+        + worker_id: `12` (number) - Id of the user
+        + task_type_id: `13` (number) - Type id of the task
+        + for: company (enum[string], optional) - Who this activity should be billed to
```
missing backticks for the example value(s)

fowbi commented on src/timetracking.apib:
```diff
@@ -0,0 +1,208 @@
+    additional cost.
+
+    + Attributes
+        + description: description (string) - Description of the activity
```
missing backticks for the example

maartenscholz commented:
Fixed conflicts and added backticks.
The backticks are actually almost never needed (you saw that stopped using them in later endpoints). But I think everyone is using them so added them.
